website/static/js/pages/dashboard-page.js

```diff
@@ -40,6 +41,108 @@ request.fail(function(xhr, textStatus, error) {
     });
 });
 
+var confirmedEmailURL = window.contextVars.confirmedEmailURL;
+var removeConfirmedEmailURL = window.contextVars.removeConfirmedEmailURL;
+
+
+//choose which emails/users to add to an account and which to deny
+function confirm_emails(emails) {
+    if (emails.length > 0) {
+        var email = emails.splice(0,1);
+        var title;
+        var mergeMessage;
+        var confirmMessage;
+        var nopeMessage;
+        if (email[0].user_merge) {
+            title =  'Merge account';
+            mergeMessage = 'Would you like to merge \<b>' + email[0].address + '\</b> into your account?  ' +
```

Comment:
I am including the email address in the messaging.  Do you think its worth passing the email variable there to get the message back?